src/Client/components/katex-mermaid.tsx

```
@@ -27,80 +27,80 @@ C -->|Two| E[Result 2]
 
-  if (className && /^language-katex/.test(className.toLowerCase())) {
-    return <code className={className} ref={refElement} />;
+  if ( className?.toLowerCase().startsWith('language-katex')) {
+    return <code className={className} ref={refElement} />
   }
 
   if (typeof children === 'string' && /^\$\$(.*)\$\$/.test(children)) {
-    const html = katex.renderToString(children.replace(/^\$\$(.*)\$\$/, '$1'), { throwOnError: false });
-    return <code dangerouslySetInnerHTML={{ __html: html }} style={{ background: 'transparent' }} />;
+    const html = katex.renderToString(children.replace(/^\$\$(.*)\$\$/, '$1'), { throwOnError: false })
+    return <code dangerouslySetInnerHTML={{ __html: html }} style={{ background: 'transparent' }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)